core/src/main/java/bio/terra/pearl/core/dao/search/EnrolleeSearchExpressionDao.java

```diff
@@ -65,11 +79,16 @@ public List<EnrolleeSearchExpressionResult> executeSearch(EnrolleeSearchQueryBui
             org.jooq.Query jooqQuery = search.toQuery(DSL.using(SQLDialect.POSTGRES), opts);
 
             Query query = jdbiFromJooq(jooqQuery, handle);
-            return query
+            StopWatch stopWatch = new StopWatch();
+            stopWatch.start();
+            var result = query
                     .registerRowMapper(Family.class, BeanMapper.of(Family.class, "family"))
                     .registerRowMapper(EnrolleeSearchExpressionResult.class, new EnrolleeSearchResultMapper())
                     .reduceRows(new EnrolleeSearchResultReducer())
                     .toList();
+            stopWatch.stop();
+            log.info("Search took {} ms", stopWatch.getTotalTimeMillis());
+            return result;
```

Comment:
Is this for testing or are we logging to keep track over time?

Reply:
ah, good catch, yeah, I wanted to make sure I wasn't unduly impacting performance.  But I can take it out